.github/workflows/frontend-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Frontend Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - '**/frontend/**'
+    - '!frontend/test/e2e/**'
```

Comment:
This line will make more sense when we merge (cummulative) changes that led to https://github.com/metabase/metabase/pull/15900 back into `master`.